src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -985,7 +994,7 @@ void network::allocate_primitive_instance(program_node const& node) {
     }
 
     _primitives[node.id()] = inst;
-    if (node.is_input()) {
+    if (node.is_type<input_layout>()) {
```

Comment:
This means that, the original impl which stored all constant inputs to _inputs were redundant?

Reply:
(Asking becuase I was curious about it before :) )

Reply:
Yeah, Looking at the usage of this _inputs, I haven't found any reason to store constants here